ereol_video/ereol_video.file_default_displays.inc

```diff
@@ -64,9 +66,45 @@ function ereol_video_file_default_displays() {
   $file_display->name = 'video__video_show__file_field_media_large_icon';
   $file_display->weight = -42;
   $file_display->status = FALSE;
-  $file_display->settings = '';
+  $file_display->settings = array(
+    'image_style' => '',
+  );
   $export['video__video_show__file_field_media_large_icon'] = $file_display;
 
+  $file_display = new stdClass();
+  $file_display->api_version = 1;
+  $file_display->name = 'video__video_show__media_videotool_image';
+  $file_display->weight = 0;
+  $file_display->status = FALSE;
+  $file_display->settings = array(
+    'image_style' => '',
```

Comment:
No image style?

Reply:
No. It's a video, not an image. The other video formats (youtube and vimeo) don't have image styles either.